src/TestcafeGherkinBootstrapper.js

```diff
@@ -146,6 +145,15 @@ module.exports = class TestcafeGherkinBootstrapper extends TestcafeBootstrapper
     );
   }
 
+  _shouldRunStep(stepDefinition, step) {
+    if (typeof stepDefinition.pattern === 'string') {
+      return [stepDefinition.pattern === step.text, []];
+    } else if (typeof stepDefinition.pattern.exec === 'function') {
```

Comment:
I googled a bit and I think `else if (stepDefinition.pattern instanceof RegExp)` is a better test for this.